{"public_date": "2014-07-02T00:00:00", "name": "CVE-2014-3533", "acknowledgement": "\nRed Hat would like to thank D-Bus upstream for reporting this issue.\n    ", "package_state": [{"fix_state": "Will not fix", "product_name": "Red Hat Enterprise Linux 5", "package_name": "dbus", "cpe": "cpe:/o:redhat:enterprise_linux:5"}, {"fix_state": "Will not fix", "product_name": "Red Hat Enterprise Linux 6", "package_name": "dbus", "cpe": "cpe:/o:redhat:enterprise_linux:6"}, {"fix_state": "Will not fix", "product_name": "Red Hat Enterprise Linux 7", "package_name": "dbus", "cpe": "cpe:/o:redhat:enterprise_linux:7"}], "document_distribution": "Copyright \u00a9 2016 Red Hat, Inc. All rights reserved.", "upstream_fix": "dbus 1.6.22, dbus 1.8.6", "threat_severity": "Moderate", "bugzilla": {"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1114416", "description": "\nCVE-2014-3533 dbus: denial of service when forwarding invalid file descriptors\n    ", "id": "1114416"}, "details": ["\ndbus 1.3.0 before 1.6.22 and 1.8.x before 1.8.6 allows local users to cause a denial of service (disconnect) via a certain sequence of crafted messages that cause the dbus-daemon to forward a message containing an invalid file descriptor.\n    "], "cvss": {"cvss_scoring_vector": "AV:L/AC:L/Au:N/C:N/I:N/A:P", "status": "draft", "cvss_base_score": "2.1"}}